### PR TITLE
Update for consistency with Objective-C sample

### PR DESCRIPTION
**Pull request by ceaglest**
* Comments, pragma marks and code organization
* Rename toggleView to be more descriptive
* Log the participant when joining a Room
* Fix adding audio multiple times

### CODE REVIEW

**Comment by ceaglest** [comment]
One thing that is missing from this PR is making the token fetching async. Could be done in another PR IMHO.

File: VideoQuickStart/ViewController.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ import UIKit
 
 import TwilioVideo
 
-class ViewController: UIViewController, UITextFieldDelegate {
+class ViewController: UIViewController {
```
Comment (ceaglest) [inline VideoQuickStart/ViewController.swift]: It was implemented in its own extension.

File: VideoQuickStart/ViewController.swift [hunk 2]
```diff
@@ -153,29 +116,69 @@ class ViewController: UIViewController, UITextFieldDelegate {
     }
     
     @IBAction func toggleMic(sender: AnyObject) {
-        if ((self.localMedia?.audioTracks.count)! > 0) {
-            self.localMedia?.audioTracks[0].isEnabled = !(self.localMedia?.audioTracks[0].isEnabled)!
+        if (self.localAudioTrack != nil) {
+            self.localAudioTrack?.isEnabled = !(self.localMedia?.audioTracks[0].isEnabled)!
```
Comment (ceaglest) [inline VideoQuickStart/ViewController.swift]: Oops, will update.

**Comment by ptankTwilio** [comment]
👍

**Pull request merged**